pkgs/by-name/op/opcua-commander/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  typescript,
+  esbuild,
+  makeWrapper,
+  nodejs,
+}:
```

```suggestion
{ lib
, buildNpmPackage
, fetchFromGitHub
, typescript
, esbuild
, makeWrapper
, nodejs
}:
```
See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax:

> but if they don't fit on one line they're written as:
> 
> { arg1, arg2, arg3
> , arg4, ...
> , # Some comment...
>   argN
> }: